apps/datafeeder/src/app/presentation/pages/dataset-validation-page/dataset-validation-page.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core'
+import { Subscription } from 'rxjs'
+import { ActivatedRoute, Router } from '@angular/router'
+import { LogService } from '@lib/common'
+
+@Component({
+  selector: 'app-dataset-validation-page',
+  templateUrl: './dataset-validation-page.html',
+  styleUrls: ['./dataset-validation-page.css'],
+})
+export class DatasetValidationPageComponent implements OnInit {
+  encodingList = [
+    {
+      label: 'UTF8',
+      value: 'UTF8',
+    },
+  ]
+  refSystem = [
+    {
+      label: 'Lambert 93',
+      value: 'Lambert93',
+    },
+  ]
+  private routeParamsSub: Subscription
```

Comment:
The goal of the `Subscription` object is mostly to unsubscribe on component `destroy()`